modules/cudev/include/opencv2/cudev/grid/reduce_to_vec.hpp

```diff
@@ -183,12 +183,14 @@ __host__ void gridReduceToColumn_(const SrcPtr& src, GpuMat_<ResType>& dst, cons
     CV_Assert( getRows(mask) == rows && getCols(mask) == cols );
 
     dst.create(1, rows);
+    cuda::createContinuous(rows, 1, dst.type(), dst);
```

Comment:
But, it is mentioned that the last param will change only if it has the same type and size. What does that mean? Should I create it and then createContinuous?

http://docs.opencv.org/2.4/modules/gpu/doc/data_structures.html#gpu-createcontinuous